src/components/menu/examples/menu-open-bottom.tsx

```diff
@@ -0,0 +1,26 @@
+import { MenuItem, ListSeparator } from '@limetech/lime-elements';
+import { Component, h } from '@stencil/core';
+
```

Comment:
@adrianschmidt do you like the commit title, message, and breaking change?

Reply:
I've updated the breaking change message, but I think I'll have some more changes to make still.

Reply:
I guess, tell me when you're done with changing the code, and I can have a look at the commits @LucyChyzhova @Kiarokh

Reply:
That's super strange, because we pulled your changes before force pushing :/ No idea what happened. Sorry for that. We're done messing around now. You can review/update again.

Reply:
No worries. I guess it's because it was just a change in the commit message, not in the code, and git really only tries to ensure that changes to the _code_ aren't lost… 🙂

Reply:
And, great work! 👏

Reply:
> But it's a little worrying that someone seems to force push things without noticing what gets lost… 😐

yeah, we were trying to always pull first 🤞.. but it could be my fault 🙈

>I guess, tell me when you're done with changing the code, and I can have a look at the commits @LucyChyzhova @Kiarokh

no more changes 😁 in the lime-elements repo